cmd/end2endtest/helpers.go

```diff
@@ -322,17 +356,17 @@ func (t *e2eElection) overwriteVote(choices []int, indexAcct int, waitType strin
 		if err != nil {
 			// check the error expected for overwrite with waitUntilNextBlock
 			if strings.Contains(err.Error(), "overwrite count reached") {
-				log.Infof("error expected: %s", err.Error())
+				log.Debugw("error expected: %s", err.Error())
 			} else {
 				return 0, errors.New("expected overwrite error")
 			}
 		}
 		contextDeadlines += ctxDeadLine
 		switch waitType {
-		case nextBlock:
```

Comment:
this looks totally unrelated to "add ballot test", is this a fix for vote overwrite? in that case please make a separate PR

Reply:
yeah no problem

Reply:
PR created https://github.com/vocdoni/vocdoni-node/pull/943